pkgs/shells/essh/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchgit, readline, autoconf, automake, libtool, yacc }:
+
+let
+  version = "fdf29d";
```

Comment:
we prefer dates over git versions, not sure where the docs really are for that though.